docs/reference/content/index.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "MongoDB Scala Driver"
+type = "index"
++++
+
+## MongoDB Scala Driver Documentation
+
+Welcome to the MongoDB Scala driver documentation hub.
+
+### Getting Started
+
+The [Getting Started]({{< relref "getting-started/index.md" >}}) guide contains installation instructions
+and a simple tutorial to get up  and running quickly.
+
+
+{{% note %}}
+This implementation is built upon the MongoDB Async Driver and mirrors the official API. 
+
+The Scala driver builds upon the Java library and provides a native feeling driver, whilst using the core driver to have the latests MongoDB Support.
```

Comment:
whilst?  Britishism.. :)

This sentence needs rewording: the phrase "latests MongoDB support" is not going to make sense to most people, IMO, though I know what you mean. How about something like:

  The MongoDB Scala driver provides an idiomatic Scala API that is built on top of the MongoDB Java driver.